openstack/imageservice/v2/images/results.go

```diff
@@ -0,0 +1,404 @@
+package images
+
+import (
+	"fmt"
+	"io"
+	"reflect"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/rackspace/gophercloud"
+	"github.com/rackspace/gophercloud/pagination"
+)
+
+// Image model
+// Does not include the literal image data; just metadata.
+// returned by listing images, and by fetching a specific image.
+type Image struct {
```

Comment:
The source code where these fields are defined:
* [types.go](https://github.com/sameo/gophercloud/blob/master/openstack/imageservice/v2/images/types.go)
* [requests.go] (https://github.com/sameo/gophercloud/blob/master/openstack/imageservice/v2/images/requests.go)

I also added some comments for all the Image structure fields. Was that what you were looking for ?